cloudhub-1/modules/ROOT/pages/dedicated-load-balancer-tutorial.adoc

```diff
@@ -35,17 +35,20 @@ Dedicated load balancers enable you to:
 * Configure proxy rules that map your applications to custom domains.
 +
 This enables you to host your applications under a single domain.
+* Enable HTTP Streaming via the API in CloudHub.
++
+[NOTE]
+Despite enabling HTTP Streaming, the uploaded file size cannot be over 200MB.
```

Comment:
Suggestion: NOTE: The uploaded file size cannot be over 200MB. Personally I'd remote the beginning of the sentence. But if you think it's necessary, it's okay.